bitherj/src/main/java/net/bither/bitherj/api/PushTxThirdParty.java

```diff
@@ -55,12 +55,6 @@ public static PushTxThirdParty getInstance() {
     }
 
     public void pushTx(Tx tx) {
-        String raw = Utils.bytesToHexString(tx.bitcoinSerialize());
```

Comment:
this first line can keep

Reply:
also add comment below
// Transaction push disabled for Bitcoin explorers, e.g. pushToBlockChainInfo()